spec/factories.rb

```diff
@@ -1,2 +1,11 @@
 FactoryGirl.define do
+  factory :author do
+    birth_country 'United States'
+    gender 'Female'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.